README.rst

```diff
@@ -6,5 +6,15 @@ in a journal or on the internet. With a short Python script and an intuitive
 model-building syntax you can design directed and undirected graphs and save
 them in any formats that matplotlib supports.
 
-Get more information at: `daft-pgm.org <http://daft-pgm.org>`_
+Get more information at `daft-pgm.org <http://daft-pgm.org>`_
+
+Try making some PGMs with the `example notebooks <http://mybinder.org:/repo/drphilmarshall/daft>`_
```

Comment:
@ixkael I still see drphilmarshall URLs... but you must be so close to that Kwak by now! :-)